### DISCUSSION TITLE
Javascript SDK - toMetaplexFileFromBrowser is not a function ERROR

### DISCUSSION THREAD

**Discussion by BushraHussain** [metaplex-foundation/js#255] [Q&A] [answered]
Hi, I'm working on creating NFT using Metaplex javascript SDK. I need to upload image and video to Arweave (using Bundler storage). I want to use metaplex file type inside SDK. I tried all below functions. Getting same error for all functions
1. toMetaplexFileFromJson
2. toMetaplexFile
3. toMetaplexFile
4. toMetaplexFileFromBrowser

Same error, toMetaplexFileFromBrowser is not a function 
--------------------
import { toMetaplexFileFromBrowser,  toMetaplexFileFromJson, toMetaplexFile } from "@metaplex-foundation/js";

const file = toMetaplexFileFromJson({ foo: 42 });

const file = toMetaplexFile('The content of my file', 'my-file.txt');

const buffer = fs.readFileSync('/path/to/my-file.txt');
const file = toMetaplexFile(buffer, 'my-file.txt');

const browserFile: File = event.target.files[0];
const file: MetaplexFile = await toMetaplexFileFromBrowser(browserFile);

![Screenshot from 2022-08-24 17-05-17](https://user-images.githubusercontent.com/11967420/186416163-e5f301be-123d-4339-b326-0d171ab0a55b.png)

Anyone can help?
Thanks in advance

**Comment by lorisleiva** [comment]
Hi there 👋 

Webpack is a bit of a tricky one with crypto-related packages and may require some extra settings.

Did you check the `js-examples` repository for getting started with Webpack 4 or 5?

https://github.com/metaplex-foundation/js-examples

**Reply by BushraHussain** [reply]
Hi thanks for response. 
Yeah i'm using same example repository and tried same code in connectWallet example and getting started with express (in createNFT.cjs) . Getting same error. TypeError: toMetaplexFileFromBrowser is not a function 
![Screenshot from 2022-08-24 20-10-45](https://user-images.githubusercontent.com/11967420/186455392-8372722c-8706-45c2-92c3-b4a08e429599.png)

**Reply by lorisleiva** [reply]
I'm afraid I've just tried with both and I can't reproduce. Would you be able to share a public repository with the minimum steps required to reproduce this issue? That way I can clone it and see what's going on.

**Reply by BushraHussain** [reply]
Yeah sure thanks. I'll create and share in a day

**Reply by BushraHussain** [reply]
Hi @lorisleiva .
Here's the link to public repo https://github.com/BushraHussain/metaplex-js-sdk
I created a new js module "MintNFT" inside pages folder. Please check and provide your feedback
Thanks in advance

**Comment by lorisleiva** [comment, marked as answer]
The problem was that you were not using the latest version of the SDK.

Here's a fix: https://github.com/BushraHussain/metaplex-js-sdk/pull/1

**Reply by BushraHussain** [reply]
Hi @lorisleiva Thank you so much for help. I just tested and it is working :) 
More prayers for you and your team. Stay blessed
